src/Yoti.Auth/ShareUrl/Extensions/DeviceLocation.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Newtonsoft.Json;
+
+namespace Yoti.Auth.ShareUrl.Extensions
+{
+    public class DeviceLocation
+    {
+        [JsonProperty(PropertyName = "latitude")]
+        private readonly double _latitude;
+
+        [JsonProperty(PropertyName = "longitude")]
+        private readonly double _longitude;
+
+        [JsonProperty(PropertyName = "radius")]
+        private readonly double _radius;
+
+        [JsonProperty(PropertyName = "max_uncertainty_radius")]
+        private readonly double _maxUncertainty;
+
+        public DeviceLocation(double latitude, double longitude, double radius, double maxUncertainty)
+        {
+            _latitude = latitude;
+            _longitude = longitude;
+            _radius = radius;
+            _maxUncertainty = maxUncertainty;
+        }
+
+        public double Latitude
```

Comment:
Perhaps we should make these getters private (or remove them) as we only need to build and serialise to JSON? (Same would apply to other SDKs)